fischkapp/src/App.tsx

```diff
@@ -9,16 +8,24 @@ import { Header } from "./components/Header";
 import { GlobalStyle } from "./components/styles/GlobalStyles";
 import { useState } from "react";
 import { flashCardsData } from "./data/data";
-
-interface FlashCardType {
-  id: number;
-  frontText: string;
-  backText: string;
-}
+import axios from "axios";
+import { useEffect } from "react";
 
 function App() {
   const [isNewCard, setNewCard] = useState(false);
-  const [flashCards, setFlashCards] = useState<FlashCardType[]>(flashCardsData);
+  const [flashCards, setFlashCards] = useState<FlashCardType[]>([]);
+
+  const getFlashCards = () => {
+    const url = "https://training.nerdbord.io/api/v1/fischkapp/flashcards";
```

Comment:
**suggestion:** You will need this URL later as well. I suggest you to create some `api` directory, where you will keep `getFlashcards`, and other API related method